include/ignition/rendering/base/BaseGpuRays.hh

```diff
@@ -200,6 +203,9 @@ namespace ignition
       /// \brief Number of channels used to store the data
       protected: unsigned int channels = 1u;
 
+      /// \brief Scanning Pattern
+      protected: std::string scanningPattern = "avia";
```

Comment:
Let's make this an enum, and the SetScanningPattern function takes in the enum as an argument.